### PR TITLE
add: properties related to border

### PR DESCRIPTION
**Pull request by nojiritakeshi**
The following styles were added.
```
  borderX: "border-left,border-right",
  borderY: "border-top,border-bottom",
  borderTopLeftRadius: "border-top-left-radius",
  borderTopRightRadius: "border-top-right-radius",
  borderBottomLeftRadius: "border-bottom-left-radius",
  borderBottomRightRadius: "border-bottom-right-radius",
  borderTopStyle: "border-top-style",
  borderBottomStyle: "border-bottom-style",
  borderLeftStyle: "border-left-style",
  borderRightStyle: "border-right-style",
  borderTopWidth: "border-top-width",
  borderBottomWidth: "border-bottom-width",
  borderLeftWidth: "border-left-width",
  borderRightWidth: "border-right-width",
  borderStart: "border-inline-start",
  borderEnd: "border-inline-end",
  borderStartWidth: "border-inline-start-width",
  borderEndWidth: "border-inline-end-width",
  borderStartStyle: "border-inline-start-style",
  borderEndStyle: "border-inline-end-style",
  borderStartRadius: "border-top-left-radius,border-bottom-left-radius",
  borderEndRadius: "border-top-right-radius,border-bottom-right-radius",
```

### CODE REVIEW

File: packages/system/src/border.ts [hunk 1]
```diff
@@ -1,12 +1,87 @@
...
+     */
+    borderEndWidth: CSSValue<"borderInlineEndWidth", true>;
+    /**
+     * @see borderInlineStartStyle
+     */
+    borderStartStyle: CSSValue<"borderInlineStartStyle">;
+    /**
+     * @see borderInlineEndStyle
+     */
+    borderEndStyle: CSSValue<"borderInlineEndStyle">;
```
Comment (poteboy) [inline packages/system/src/border.ts:41]: I don't believe that `borderEndStyle` should accept a numerical value. This concern also extends to some of the other styles added in this PR.

Comment (nojiritakeshi) [reply packages/system/src/border.ts:41]: @poteboy 
I don't think borderEndStyle accepts numeric values

Comment (poteboy) [reply packages/system/src/border.ts:41]: Oh, what I actually meant by that is we should filter the third argument of `applyResponsiveStyles` based on whether or not the property accepts a numerical value.

Comment (nojiritakeshi) [reply packages/system/src/border.ts:41]: I fixed what you pointed out. 
please confirm.

**Review by poteboy — APPROVED** [review]

**Comment by masa5555** [comment]
#150

**Pull request merged**